src/System.Net.Security/tests/FunctionalTests/CertificateValidationClientServer.cs

```diff
@@ -69,19 +68,19 @@ public CertificateValidationClientServer()
                     Task clientAuthentication = sslClientStream.AuthenticateAsClientAsync(
                         serverName,
                         clientCerts,
```

Comment:
is this test passing in Linux? client cert validation passes without any prerequisite step?